CodeMaid.UnitTests/CodeMaid.UnitTests.csproj

```diff
@@ -108,11 +79,20 @@
       <Link>Properties\CodeMaid.snk</Link>
     </None>
     <None Include="app.config" />
-    <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup>
-    <Analyzer Include="..\packages\Microsoft.VisualStudio.Threading.Analyzers.15.8.209\analyzers\cs\Microsoft.VisualStudio.Threading.Analyzers.CodeFixes.dll" />
-    <Analyzer Include="..\packages\Microsoft.VisualStudio.Threading.Analyzers.15.8.209\analyzers\cs\Microsoft.VisualStudio.Threading.Analyzers.dll" />
+    <PackageReference Include="Microsoft.VisualStudio.SDK">
+      <Version>17.0.0-previews-1-31410-273</Version>
```

Comment:
You could consider using [Microsoft.Build.CentralPackageVersions](https://github.com/microsoft/MSBuildSdks/blob/main/src/CentralPackageVersions/README.md) to avoid specifying same versions across multiple projects.
Here's an example of a similar migration: https://github.com/gitextensions/gitextensions/pull/9161